api/descriptions/policies/vulnerability_images_impacted_get.md

```diff
@@ -1,12 +1,114 @@
-Retrieves a list of all resources a vulnerability rule impacts.
-These rule names can be found from the `name` variable in the response from a `GET` on the basic policies/cve endpoint.
+Returns a list of impacted images where the policy is applied. This provides early insight into potential vulnerabilities caused by your repo's external dependencies long before they are deployed by CI/CD pipelines.
```

Comment:
@KICKdesigns I'm not sure the second sentence is true. I think this endpoint is mostly used by the UI guys to render charts in the UI.